BeagleBone/Black/analogInCallback.js

```diff
@@ -22,3 +22,13 @@ function printStatus(err, x) {
     process.stdout.write(inputPin + ': ' + (x*100).toFixed(1) + '%, ' + (1.8*x).toFixed(3) + 'V   \r');
     b.analogRead(inputPin, printStatus);
 }
+
+// Bone  | Pocket | AIN
```

Comment:
My thinking was the things in BeagleBone should always run out-of-the-box, thus slightly different versions for different platforms.

I put the table at the bottom since it wan't needed to run, but was there for later adaption.